humble/distribution.yaml

```diff
@@ -3791,7 +3791,7 @@ repositories:
       tags:
         release: release/humble/{package}/{version}
       url: https://github.com/ros2-gbp/rmf_demos-release.git
-      version: 2.0.0-1
+      version: 2.0.1-1
```

Comment:
@orensbruli should both `source` and `doc` versions be `humble`?

Reply:
You are right. Fixed.